src/e2e/puppeteer/__tests__/sidebar.ts

```diff
@@ -0,0 +1,46 @@
+import path from 'path'
+import configureSnapshots from '../configureSnapshots'
+import click from '../helpers/click'
+import hideVisibility from '../helpers/hideVisibility'
+import press from '../helpers/press'
+import screenshot from '../helpers/screenshot'
+import setTheme from '../helpers/setTheme'
+import type from '../helpers/type'
+
+expect.extend({
+  toMatchImageSnapshot: configureSnapshots({ fileName: path.basename(__filename).replace('.ts', '') }),
+})
+
+vi.setConfig({ testTimeout: 20000, hookTimeout: 20000 })
+
+/** Open sidebar and wait for it to slide all the way open. */
+const openSidebar = async () => {
+  await click('[aria-label=menu]')
+  await new Promise(resolve => setTimeout(resolve, 100))
```

Comment:
Rather than setting an arbitrary timeout, I think we can wait for an element in the sidebar to appear. That will generally be faster.

```ts
await page.locator('[data-testid="TEST_ID"]').wait()
```

Reply:
Thanks! Also all the failing tests are light mode related, so I'll just update them.